config/database.yml

```diff
@@ -18,11 +18,7 @@ devunicorn:
 
 test: &test
   <<: *default
-  database: <%= ENV['CBO_BASE_DATABASE_DATABASE'] || 'cbo_test' %>
```

Comment:
just removing superfluos config. the `develop` branch no longer exists but did in intial stages of the project